src/SFA.DAS.LevyTransferMatching.Web/Models/Opportunities/MoreDetailsPostRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿using SFA.DAS.LevyTransferMatching.Web.Attributes;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Models.Opportunities
+{
+    public class MoreDetailsPostRequest : ApplyRequest
+    {
+        public string EncodedPledgeId { get; set; }
+        [MaxWords(200)]
```

Comment:
Personally, I'd rather we were consistent in having all validation handled by fluent validators, rather than "data annotations" style validation

Reply:
Changed it